csrankings-m.csv

```diff
@@ -2109,6 +2109,7 @@ Mingxing Xu,Tsinghua University,http://www.cs.tsinghua.edu.cn/publish/csen/4623/
 Mingxun Wang,Univ. of California - Riverside,https://www.cs.ucr.edu/~mingxunw/,jMvg4eMAAAAJ
 Mingyan Liu,University of Michigan,http://web.eecs.umich.edu/~mingyan,WiIM-MgAAAAJ
 Mingyi He,NWPU,http://dianzi.nwpu.edu.cn/info/1269/5955.htm,gLnLpAsAAAAJ
+Mingyi Hong,University of Minnesota,https://people.ece.umn.edu/~mhong/mingyi.html,qRnP-p0AAAAJ

```

Comment:
Not CS. Please clarify how they meet inclusion criteria.

Reply:
Prof. Mingyi Hong can advise student from the CS department so he meets the inclusion criteria.

Reply:
This needs evidence.